app/templates/public.hbs

```diff
@@ -77,7 +77,8 @@
               {{/each}}
             </div>
           {{/if}}
-          <Public::SocialLinks @externalUrl={{this.model.externalEventUrl}} @socialLinks={{this.model.socialLinks}} />
+          <Public::SocialLinks @externalUrl={{this.model.externalEventUrl}} @socialLinks={{this.model.socialLinks}} @eventname={{this.model.name}} @eventlocation={{this.headerLocation}}
```

Comment:
This should not be in social links component and pass event directly instead of passing 5 props based on single event